pkg/subctl/cmd/cloud/cleanup/aws.go

```diff
@@ -23,13 +23,9 @@ import (
 	"github.com/submariner-io/submariner-operator/internal/cli"
 	"github.com/submariner-io/submariner-operator/internal/exit"
 	"github.com/submariner-io/submariner-operator/internal/restconfig"
-	cloudaws "github.com/submariner-io/submariner-operator/pkg/cloud/aws"
 	"github.com/submariner-io/submariner-operator/pkg/cloud/cleanup"
-	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/cloud/aws"
 )
 
-var config cloudaws.Config
```

Comment:
This is unused for cleanup.

Reply:
Not really. As discussed over slack, these flags are needed for `cleanup`. See changes I made for RHOS https://github.com/submariner-io/submariner-operator/pull/2048/files#diff-3561b3b903372de9ecaaaf82c597d35ca4a3a9a1dbefe02273b5bcd8519a686d

Reply:
yeah I'm fixing that in another PR.